apps/react/src/store/anime/selectors.ts

```diff
@@ -0,0 +1,21 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+import { RootState } from '../store';
+
+import { animeAdapter } from './state';
+
+const animeSelector = animeAdapter.getSelectors<RootState>(
+  state => state.anime,
+);
+
+/** Selects all anime from store. */
+export const selectAnimeList = createSelector(
+  (state: RootState) => animeSelector.selectAll(state),
+  animeList => animeList,
+);
```

Comment:
There is no need for `selectAnimeList`
```suggestion
export const { selectAll: selectAnimeList } = animeAdapter.getSelectors<RootState>(
  state => state.anime,
);
```